### PR TITLE
add "load blalncer" prefix to reports

### PR DESCRIPTION
**Pull request by haim-kermany**

### CODE REVIEW

File: pkg/ibmvpc/vpc.go [hunk 1]
```diff
@@ -244,6 +244,10 @@ type LoadBalancer struct {
 	listeners []LoadBalancerListener
 }
 
+func (lb *LoadBalancer) ExtendedName(c *vpcmodel.VPCConfig) string {
+	return lb.Kind() + ":" + lb.ExtendedPrefix(c) + lb.Name()
+}
+
```
Comment (adisos) [inline pkg/ibmvpc/vpc.go:250]: - the function can be `func (lb *LoadBalancer) Name()` instead of `func (lb *LoadBalancer) ExtendedName()` ?
- I would change the format to (example): `service-alb[LoadBalancer]`

File: pkg/ibmvpc/analysis_output_test.go [hunk 2]
```diff
@@ -656,7 +656,7 @@ func TestAllWithComparison(t *testing.T) {
 		if formatsAvoidComparison[tt.format] {
 			tt.mode = outputIgnore
 		} else {
-			tt.mode = outputComparison
+			tt.mode = outputGeneration
```
Comment (adisos) [inline pkg/ibmvpc/analysis_output_test.go:659]: why is this changed?

**Review by adisos — CHANGES_REQUESTED** [review]

File: pkg/ibmvpc/analysis_output_test.go (hunk 2 above)
Comment (haim-kermany) [reply pkg/ibmvpc/analysis_output_test.go]: sorry, fixed

File: pkg/ibmvpc/vpc.go (hunk 1 above)
Comment (haim-kermany) [reply pkg/ibmvpc/vpc.go]: -all resource has both Name() and ExtendedName() (please notice that ExtendedName() call Nam()). in Drawio We use Name() (we need it there to be short). ExtendedName() is for textual reports.

- fixed

Comment (adisos) [reply pkg/ibmvpc/vpc.go]: This is the current documentation for `ExtendedName` 
```
	// ExtendedName returns a resource name that includes its VPC as prefix when necessary.
	// for example, a subnet with name "s1" within VPC "v1" will have extended name: "v1/s1"
	// note this method is relevant only for Node and Subnet objects.
	// note it adds the prefix only for input config that has multiple VPCs context.
	ExtendedName(*VPCConfig) string
```

So this function for LB breaks this semantics..

in textual reports both `Name` and `ExtendedName` are used .
how about having a new method for VPCResource interface `ShortName` , to be used for drawio? a method that only returns the resource name ?

Comment (haim-kermany) [reply pkg/ibmvpc/vpc.go]: From the code its look like the textual report always uses ExtendedName(*VPCConfig), however, if we do not in multiple VPCs context, the ExtendedPrefix() returns ""
I updated documentation

Comment (haim-kermany) [reply pkg/ibmvpc/vpc.go]: fixed
ShortName() is not needed, for LB in drawio,  I just use the resourceName

Comment (adisos) [reply pkg/ibmvpc/vpc.go]: opened issue #590

**Review by adisos — APPROVED** [review]
